cmd/minishift/cmd/openshift/set.go

```diff
@@ -37,8 +37,8 @@ const (
 	patchFlag  = "patch"
 
 	unknownPatchTargetError = "Unkown patch target. Only 'master' and 'node' are supported."
-	emptyPatchError         = "You need to specify a patch using --patch."
-	invalidJSONError        = "The specified patch need to be valid JSON."
+	emptyPatchError         = "You must specify a patch using the --patch flag."
+	invalidJSONError        = "You must specify a patch with a valid JSON format."
```

Comment:
The article 'a' isn't a quantifier in this case, rather it makes it explicit that the JSON format needs to be valid. However, to help be even more explicit, we can write something like "a valid JSON file". Simply writing "valid JSON" is incomplete grammatically and needs a qualifier.